variables.tf

```diff
@@ -615,6 +615,12 @@ variable "gitlab_runner_registration_config" {
   }
 }
 
+variable "secure_parameter_store_registration_token_key" {
```

Comment:
suggestion: rename to something like `secure_parameter_store_gitlab_runner_registration_token_name`. `gitlab_runner` as it is the GitLab Runner registration token, `name` as this is used by `aws ssm` and in the AWS Console. What do you think?